scanner/src/main/kotlin/storages/PostgresStorage.kt

```diff
@@ -90,11 +96,20 @@ class PostgresStorage(
             defaultFetchSize(1000)
         }
 
+        var hik = dataSource as HikariDataSource
+        var config = hik.getHikariConfigMXBean() as HikariConfig
+        val schema = Schema(config.schema)
+
         transaction {
-            withDataBaseLock {
+            withDataBaseLock {                              
```

Comment:
Does this line contain trailing whitespace? It looks like so. These should be removed.

Reply:
Fixed lints